### PR TITLE
Add bls_undefined model to model repo and add to c_api test

### PR DESCRIPTION
**Pull request by debermudez**

### CODE REVIEW

File: qa/python_models/bls_undefined/config.pbtxt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: "bls_undefined"
```
Comment (Tabrizian) [inline qa/python_models/bls_undefined/config.pbtxt:1]: Copyright is missing

File: qa/python_models/bls_undefined/model.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import triton_python_backend_utils as pb_utils
```
Comment (Tabrizian) [inline qa/python_models/bls_undefined/model.py:1]: Copyright is missing here too.

File: qa/L0_perf_analyzer_capi/test.sh [hunk 3]
```diff
@@ -288,6 +293,18 @@ if [ $(cat $CLIENT_LOG | grep "${NON_SUPPORTED_ERROR_STRING}" | wc -l) -ne 1 ];
 fi
 set -e
 
+# Testing erroneous configuration
+# This model is expected to fail
+$PERF_ANALYZER -v -m bls_undefined --shape INPUT0:1048576 -t 64\
```
Comment (Tabrizian) [inline qa/L0_perf_analyzer_capi/test.sh:298]: You need to update copyright in this file too.

File: qa/python_models/bls_undefined/config.pbtxt (hunk 1 above)
Comment (debermudez) [reply qa/python_models/bls_undefined/config.pbtxt:27]: done

File: qa/python_models/bls_undefined/model.py (hunk 2 above)
Comment (debermudez) [reply qa/python_models/bls_undefined/model.py:27]: done

File: qa/L0_perf_analyzer_capi/test.sh (hunk 3 above)
Comment (debermudez) [reply qa/L0_perf_analyzer_capi/test.sh:298]: done

**Review by Tabrizian — APPROVED** [review]

**Pull request merged**